plugins/catalog-react/src/filters.ts

```diff
@@ -94,6 +94,7 @@ export class EntityTextFilter implements EntityFilter {
     const partialMatch = this.toUpperArray([
       entity.metadata.name,
       entity.metadata.title,
+      (entity.spec?.profile as { displayName?: string })?.displayName,
```

Comment:
I don't think this change is needed as paginated tables don't rely on this method.

```suggestion
```

Reply:
@vinzscam You are right it seems that this one can be removed 🤔 Is there a way to use non-paginated table ? Should we keep it for non-paginated table ?